build/automation/etc/githooks/scripts/python-code-pre-commit.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+exit 0
```

Comment:
Is there a reason behind this?

Reply:
This is what Dan added to ignore this step whilst it's non-functional